internal/stack/resources.go

```diff
@@ -115,6 +115,11 @@ var (
 			Path:    ElasticAgentEnvFile,
 			Content: staticSource.Template("_static/elastic-agent.env.tmpl"),
 		},
+		&resource.File{
+			Path:         "logstash_startup.sh",
+			CreateParent: true,
+			Content:      staticSource.Template("_static/logstash_startup.sh"),
+		},
```

Comment:
If we need this file to be executable we can also do it here:
```suggestion
			Mode: resource.FileMode(0755),
		},
```

Reply:
Ah, thanks @jsoriano, this must be something I was looking for 🤦 ... Logically/technically the permission should be inherited same as a host file has... let's see.